flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/stream/sql/MiniBatchIntervalInferTest.xml

```diff
@@ -16,6 +16,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <Root>
+  <TestCase name="testMiniBatchOnDataStreamWithRowTime">
+    <Resource name="sql">
+      <![CDATA[
+SELECT long,
+  COUNT(str) as cnt,
+  TUMBLE_END(rowtime, INTERVAL '10' SECOND) as rt
+FROM T1
+GROUP BY long, TUMBLE(rowtime, INTERVAL '10' SECOND)
+      ]]>
+    </Resource>
+    <Resource name="planBefore">
+      <![CDATA[
+LogicalProject(long=[$0], cnt=[$2], rt=[TUMBLE_END($1)])
++- LogicalAggregate(group=[{0, 1}], cnt=[COUNT($2)])
+   +- LogicalProject(long=[$0], $f1=[TUMBLE($3, 10000:INTERVAL SECOND)], str=[$2])
+      +- LogicalTableScan(table=[[default_catalog, default_database, T1]])
+]]>
+    </Resource>
+    <Resource name="planAfter">
+      <![CDATA[
+Calc(select=[long, cnt, w$end AS rt])
++- GroupWindowAggregate(groupBy=[long], window=[TumblingGroupWindow('w$, rowtime, 10000)], properties=[w$start, w$end, w$rowtime, w$proctime], select=[long, COUNT(str) AS cnt, start('w$) AS w$start, end('w$) AS w$end, rowtime('w$) AS w$rowtime, proctime('w$) AS w$proctime])
+   +- Exchange(distribution=[hash[long]])
+      +- Calc(select=[long, rowtime, str])
+         +- DataStreamScan(table=[[default_catalog, default_database, T1]], fields=[long, int, str, rowtime])
```

Comment:
Nope. This is because we don't support minibatch window yet. We don't do any mini-batch for window aggregates for now, so there is no MiniBatchAssigner. This will be introduced when we support minibatch window. see [MiniBatchIntervalInferRule.scala#L72](https://github.com/apache/flink/blob/master/flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/physical/stream/MiniBatchIntervalInferRule.scala#L72)